.github/workflows/ci-tests.yml

```diff
@@ -50,10 +53,7 @@ jobs:
     - name: Checkout Repo
       uses: actions/checkout@v3
     - name: Run Tuist Generation
-      uses: tuist/tuist-action@0.13.0
-      with:
-          command: 'generate'
-          arguments: ''
+      uses: ./.github/actions/run-tuist-generation
```

Comment:
We already had a local action but it wasn't being used. This drops the broken tuist action from the marketplace and uses our own one that we can control.